src/steamship/data/block.py

```diff
@@ -79,6 +79,11 @@ def query(
         )
         return res
 
+    def index(self, plugin_instance: Any = None):
```

Comment:
```suggestion
    def index(self, embedding_plugin_instance: EmbeddingIndex = None):
```

This only works for embedding_plugin_instances right?

Reply:
Yes -- will update w/ this suggestion. Thanks!